apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.py

```diff
@@ -156,15 +184,13 @@ def render(element_html: str, data: pl.QuestionData) -> str:
 
         html_params["question"] = data["panel"] == "question"
         with open("pl-rich-text-editor.mustache", encoding="utf-8") as f:
-            html = chevron.render(f, html_params).strip()
+            return chevron.render(f, html_params).strip()
 
     elif data["panel"] == "answer":
-        html = ""
+        return ""
     else:
         raise ValueError("Invalid panel type: " + data["panel"])
```

Comment:
```suggestion
        assert_never(data["panel"])
```
Somewhat tangential but this section has some cleanup in it anyway.